socs/agents/smurf_file_emulator/agent.py

```diff
@@ -521,6 +521,7 @@ def uxm_setup(self, session, params):
             time.sleep(1)
         return True, "Wrote tune files"
 
+    @ocs_agent.param('tag', default=None)
```

Comment:
Add to the method signature override in first line of docstring. (This also made me notice the pysmurf-controller's method signatures are out of date as well. Going to file an issue for that.)

Same goes for each 'tag' arg addition.

Reply:
Ok, sorry I didn't get to this until now but just updated!